broker.go

```diff
@@ -81,35 +79,36 @@ func (b *Broker) continuousQueryLoop(done chan struct{}) {
 }
 
 func (b *Broker) runContinuousQueries() {
-	next := 0
-	for {
-		// if the current node hasn't been set it's our first time or we're reset. move to the next one
-		if b.currentCQProcessingNode == nil {
-			dataNodes := b.Broker.Replicas()
-			if len(dataNodes) == 0 {
-				return // don't have any nodes to try, give it up
-			}
-			next = next % len(dataNodes)
-			b.currentCQProcessingNode = dataNodes[next]
-			next++
-		}
+	topic := b.Broker.Topic(BroadcastTopicID)
+	if topic == nil {
+		log.Println("broker cq: no topics currently available.")
+		return // don't have any topics to get data urls from, give it up
+	}
+	dataURLs := topic.DataURLs()
```

Comment:
We should probably add a convenience method to the broker which is something like:

```go
func (b *Broker) DataURLs() ([]url.URL, bool)
```

And then we can hide some of this searching around.

There is no other way that a broker will know about data urls right now.